libraries/stdlib/js/src/org.w3c/org.w3c.dom.encryptedmedia.kt

```diff
@@ -178,6 +178,7 @@ public inline fun MediaEncryptedEventInit(initDataType: String? = "", initData:
 }
 
 /* please, don't implement this interface! */
+@JsName("null")
```

Comment:
This is a clever hack! But these files are autogenerated and fix should land in dukat. @Schahen could you help?

Reply:
Fixed. Declaration regenerated by Dukat `0.5.8-rc.4` (new)